NEWS.md

```diff
@@ -1,3 +1,11 @@
+# rlistings 0.1.2.0
+ * `cols` argument renamed to `disp_cols` in `as_listing`
+ * `as_listing` gains `non_disp_cols` argument 
```

Comment:
similar to the above comment, use "non_display_cols" rather than "non_disp_cols"

@Melkiades @edelarua @ayogasekaram @telepath37 @clarkliming , any thoughts?

Reply:
non_display_cols is getting a little long for an argument name in my opinion, which is why I didn't want to do that.

Reply:
how about 'hide_cols' or 'drop_cols' if non_display_cols is too long :)

Reply:
I kind like the gramma that dplyr provides: when you use select(var), you get the column var; when use select(-var), you get the var removed (others are still there)

Reply:
drop_cols is misleading, we are talking about whether the columns re **displayed** or not, they are still present in the underlying data. hide_cols is a verb (action), which isn't correct here. The correct-in-english related name would be hidden_cols, but that doesn't really seem right. 

Anyway this discussion belongs in a separate issue. Please file one if you feel strongly enough about this that you want to discuss it further @shajoezhu 

@clarkliming none of the `rtables` family of packages employ nonstandard evaluation (NSE) which is what select(var), select(-var) are doing. This was an intentional design choice early in the process due to the fact that in `rtables`, layout construction is considered **pre-data**, and NSE primarily only makes sense int he context of operating on or within a particular dataset.